nuxeo-features/nuxeo-platform-collections/nuxeo-platform-collections-core/src/main/java/org/nuxeo/ecm/collections/core/versioning/NoVersioningCollectionPolicyFilter.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *  
+ * Contributors:
+ *     Kevin Leturc <kleturc@nuxeo.com>
+ */
+package org.nuxeo.ecm.collections.core.versioning;
+
+import java.util.List;
+
+import org.nuxeo.ecm.collections.api.CollectionConstants;
+import org.nuxeo.ecm.collections.core.adapter.CollectionMember;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.versioning.VersioningPolicyFilter;
+
+/**
+ * Policy filter which disables automatic versioning for add/remove to/from collection actions.
+ *
+ * @since 9.2
+ */
+public class NoVersioningCollectionPolicyFilter implements VersioningPolicyFilter {
+
+    @Override
+    public boolean test(DocumentModel previousDocument, DocumentModel currentDocument) {
+        // exclude 'Collection' facet, we don't want to auto version these documents
+        if (currentDocument.hasFacet(CollectionConstants.COLLECTION_FACET)) {
+            return true;
+        }
```

Comment:
Shouldn't we return _false_ if we want to exclude the 'Collection' facet from auto-versioning ?